src/components/accordion-with-image/accordionInfo.js

```diff
@@ -0,0 +1,25 @@
+import MapLogo from '~/img/guest-home-page/map.svg'
+
+const accordionInfo = [ 
+  {
+    image: MapLogo,
+    heading:'accordion.flexibleLocation.heading', 
+    subHeading:'accordion.flexibleLocation.subHeading',
+  },
+  {
+    image: MapLogo,
+    heading:'accordion.individualTime.heading', 
+    subHeading:'accordion.individualTime.subHeading',
+  },
+  {
+    image: MapLogo,
+    heading:'accordion.freeChoiceOfTeachers.heading', 
+    subHeading:'accordion.freeChoiceOfTeachers.subHeading',
+  },
+  {
+    image: MapLogo,
+    heading:'accordion.digitalCommunication.heading', 
+    subHeading:'accordion.digitalCommunication.subHeading',
+  }
+]
+export default accordionInfo

```

Comment:
export not as default, use const export. export const accrordionInfo.. and rename accordionInfo. Its more like descriptionTimes

Reply:
++